src/SearchResults/index.js

```diff
@@ -420,6 +420,8 @@ SearchResults.prototype.getFacetByName = function(name) {
     find(this.hierarchicalFacets, predicate);
 };
 
+var EMPTY_NORMALIZED_FACET_VALUES = (function() { var a = []; a.facetType = 'conjunctive'; return a; })();
```

Comment:
But the documentation states that this values is attached when returned from extractNormalizedFacetValues, but then we should probably remove it from the returned array.